windows/container.go

```diff
@@ -90,16 +83,16 @@ func (c *container) Start(ctx context.Context) error {
 	}
 
 	c.setStatus(runtime.RunningStatus)
-	c.sendEvent(c.ctr.ID(), events.RuntimeEvent_START, c.ctr.Pid(), 0, time.Time{})
+	//	c.sendEvent(c.ctr.ID(), events.RuntimeEvent_START, c.ctr.Pid(), 0, time.Time{})
```

Comment:
Ya, I just commented these out until we finished the windows work and have it running again